CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/BaseTest/AccessibilityIdentifier/ConfigurationScreenAccessibilityIdentifier.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ConfigurationScreenAccessibilityIdentifier.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+enum ConfigurationScreenAccessibilityIdentifier:String{
```

Comment:
[suggestion] Let's stick with Settings instead of Configuration. We should not introduce a new name for this screen. Let's avoid difficult conversations where we are using two names for the same thing.
Can you please everywhere where this Configuration is introduced? Thank you!

Reply:
Totally agreed, I can change the names to Settings.